test/e2e/infra/vagrant/playbook/roles/common/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 - name: "Checking that Ansible version >= '2.4.0'"
   assert:
-    that: ansible_version.full >= "2.4.0"
```

Comment:
I see the doc says version_compared was renamed to version in 2.5.0. So does it still work if using 2.5.0+?
https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#comparing-versions

cc @antoninbas if he has any idea.

Reply:
Just saw @antoninbas has tested this with 2.9.13. Then was the doc wrong? or it keeps both version and version_compared?

Reply:
@tnqn the proposed change works for me with 2.9. It seems that as of today, even top-of-tree supports both: https://github.com/ansible/ansible/blob/221c50b57c347d6f8382523c48e869cb21b8c010/lib/ansible/plugins/test/core.py#L256-L257

Reply:
My local machine is ansible 2.10.3 and it seems ansible still support `version_compared` at 2.10.3

Reply:
Thanks @antoninbas and @ceclinux for confirming. Then it's a doc thing, this patch added that statement but just added an alias "version", not renaming it.
https://github.com/ansible/ansible/pull/32361/files#diff-64ad78caca92056d49f0e3f1a0a3498ea24f9af508981cca7c7dfa21de4004b5R149